### ISSUE TITLE
coords for 3D

### ISSUE THREAD

**Issue by mauro3** [evetion/GeoArrays.jl#33]
Should this work:
```julia
julia> GeoArrays.coords(GeoArrays.GeoArray(rand(2,3,4)), [1,1,1])
ERROR: DimensionMismatch("expected input array of length 2, got length 3")
Stacktrace:
 [1] dimension_mismatch_fail(::Type{T} where T, ::Array{Int64,1}) at /home/mauro/.julia/packages/StaticArrays/1g9bq/src/convert.jl:18
 [2] convert at /home/mauro/.julia/packages/StaticArrays/1g9bq/src/convert.jl:23 [inlined]
 [3] StaticArray at /home/mauro/.julia/packages/StaticArrays/1g9bq/src/convert.jl:7 [inlined]
 [4] coords(::GeoArray{Float64}, ::Array{Int64,1}) at /home/mauro/.julia/dev/GeoArrays/src/geoarray.jl:52
 [5] top-level scope at REPL[225]:1
```
?

**Comment by mauro3** [comment]
Sorry, I got confused, the 3rd dim is for bands not z-values, right?   See also #34.

**Comment by visr** [comment]
> 3rd dim is for bands not z-values, right

Yeah that's right. Although I there is nothing that says you can't store z values in bands. But no standardized way within GDAL at least to then keep track of the coords as far as I know: https://gdal.org/user/raster_data_model.html#raster-band.

**Comment by evetion** [comment]
Closed, follow further discussion in #34
